src/test/kotlin/com/zopa/ktor/opentracing/KtorOpenTracingTest.kt

```diff
@@ -67,10 +68,10 @@ class KtorOpenTracingTest {
             with(mockTracer.finishedSpans()) {
                 assertThat(size).isEqualTo(1)
                 assertThat(first().parentId()).isEqualTo(0L) // no parent span
-                assertThat(first().operationName()).isEqualTo("GET /greeting/<UUID>")
+                assertThat(first().operationName()).isEqualTo("GET /greeting/{id}")
+                assertThat(first().tags().get("id")).isEqualTo("ab7ad59a-a0ff-4eb1-90cf-bc6d5c24095f")
                 assertThat(first().tags().get("span.kind")).isEqualTo("server")
                 assertThat(first().tags().get("http.status_code")).isEqualTo(200)
-                assertThat(first().tags().get("UUID")).isEqualTo("ab7ad59a-a0ff-4eb1-90cf-bc6d5c24095f")
             }
```

Comment:
in your previous PR you added some really nice assertions; https://github.com/zopaUK/ktor-opentracing/pull/20/files#diff-0f73efcc54690024e53b01dc0d290c77eceddfa347e01c69452be28d9c080e8bR426 
would you consider adding them back ? :)

Reply:
I'm assuming you mean to the `Client passes trace context in headers` test?

Reply:
Added those ones anyway

Reply:
nice! yes that's what i meant, i commented on the wrong test 🙈